data/pages-content/sprints.md

```diff
@@ -220,6 +220,24 @@ Writing PEPs for the given Issues below. The organizer has never written [a PEP]
    - [Not Operator](https://github.com/python/typing/issues/801)
    - [copy_kwargs function](https://stackoverflow.com/questions/71968447/python-typing-copy-kwargs-from-one-function-to-another)
 
+### Pytorch - Tensors and Dynamic neural networks in Python
+
+- Number of people: ~10 (1 organizer)
+- Room: RB 104
+- Python Level: 2+ years of experience recommended and some C++ experience
+- Contact: Piotr Gryko ([Email](mailto:piotr.gryko@gmail.com), [GitHub](https://github.com/pgryko))
+  [Pytorch Forum](https://discuss.pytorch.org/)
+  [pytorch Github](https://github.com/pytorch/pytorch)
+- Links:
+  - [Building from source using docker (recommended)] (https://github.com/cresset-template/cresset)
+  - [Building from source] (https://github.com/pytorch/pytorch#from-source)
+  - [Contributing docs] (https://github.com/pytorch/pytorch/blob/main/CONTRIBUTING.md)
```

Comment:
Is the space between the text inside square brackets and the link inside parenthesis intentional?

Reply:
Nope - that's a defect. I'll fix it, catch up from main, confirm that the sprint is still on the bottom and merge the PR

Thank you!

Reply:
I went ahead and did it for you, @pgryko.
If you confirm there is nothing else _you_ would like to change, I'm happy to merge.

Reply:
I'm happy with it, please go ahead and merge